Subject: fix host compile of helper tool
From: Erwin Rol <erwin@erwinrol.com>

gen_mattypes needs to be compiled for the host and not for the target

Signed-off-by: Erwin Rol <erwin@erwinrol.com>

--- Mesa-7.4.2/src/mesa/x86/Makefile.orig	2009-06-11 17:31:29.000000000 +0200
+++ Mesa-7.4.2/src/mesa/x86/Makefile	2009-06-11 17:31:53.000000000 +0200
@@ -21,7 +21,7 @@
 
 
 gen_matypes: gen_matypes.c
-	$(CC) $(INCLUDE_DIRS) $(CFLAGS) gen_matypes.c -o gen_matypes
+	$(CC_FOR_BUILD) $(INCLUDE_DIRS) $(CFLAGS_FOR_BUILD) gen_matypes.c -o gen_matypes
 
 # need some special rules here, unfortunately
 matypes.h: ../main/mtypes.h ../tnl/t_context.h gen_matypes
